generator/src/main/java/io/github/jwharm/javagi/model/Enumeration.java

```diff
@@ -15,25 +16,59 @@ public void generate(Writer writer) throws IOException {
         generateImportStatements(writer);
         generateJavadoc(writer);
 
-        writer.write("public class " + javaName + " extends io.github.jwharm.javagi.Enumeration {\n");
+        writer.write("public enum " + javaName + " implements io.github.jwharm.javagi.Enumeration {\n");
 
-        generateCType(writer);
-        generateMemoryLayout(writer);
+        List<Member> usable = memberList.stream()
```

Comment:
What is the purpose of the `usable` list in `Enumeration.generate()`? Can you please add a comment for future reference.